axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveServiceImpl.java

```diff
@@ -527,6 +532,19 @@ protected MoveLine generateReverseMoveLine(
             orgineMoveLine.getCounter(),
             orgineMoveLine.getName(),
             null);
+    reverseMoveLine = this.copyAnalytic(orgineMoveLine, reverseMoveLine);
+    return reverseMoveLine;
+  }
+
+  protected MoveLine copyAnalytic(MoveLine origineMoveLine, MoveLine reverseMoveLine) {
+    reverseMoveLine.setAnalyticDistributionTemplate(
+        origineMoveLine.getAnalyticDistributionTemplate());
+    for (AnalyticMoveLine origineAnalyticMoveLine : origineMoveLine.getAnalyticMoveLineList()) {
+      AnalyticMoveLine copyAnalyticMoveLine =
+          analyticMoveLineRepository.copy(origineAnalyticMoveLine, true);
```

Comment:
I think the parameter "deep" here needs to be false instead to avoid unnecessary copies